app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<main class="welcome-page">
+  <div class="user-login">
+    <section class='welcome-content'>
+      <%= form_for :session, url: login_path do |f| %>
+
+          <%= f.label :email %>
+          <%= f.text_field :email %><br>
+
+          <%= f.label :password %>
+          <%= f.password_field :password %><br>
+
+            <%= f.submit 'Log In' %>
+      <% end %>
+    </section>
+  </div>
+</main>
+
+
+
+  <%= button_to "Register", "/register", class: "welcome-btn" %>
```

Comment:
Do we have the link to the spotify login live here?

Reply:
This was removed - I didn't catch it when I did this commit.

Reply:
Ok just want to check.